libs/ngx-transloco-router/src/lib/localize-router.service.ts

```diff
@@ -78,7 +78,7 @@ export class LocalizeRouterService {
         let url = this.traverseRouteSnapshot(rootSnapshot);
         url = this.translateRoute(url) as string;
 
-        if (!this.settings.alwaysSetPrefix) {
+        if (this.settings.alwaysSetPrefix) {

```

Comment:
Hi, can you tell me what the current issue you are having when this is set to false?
When testing out your changes this breaks the routing when you change back and forth to different language. For example, if I am currently on 'en', change to 'fr' works fine first time, switch back to 'en' brings you to /404.

Thanks for the PR.

Reply:
Hi ! First, thank you for your work.
This is my fault, I misunderstood the utility of the alwaysSetPrefix variable.
I don't use any prefix in any language in my project, so I will use my fork.